docs/security/trust/deploying_notary.md

```diff
@@ -0,0 +1,37 @@
+<!--[metadata]>
++++
+title = "Deploying Notary"
+description = "Deploying Notary"
+keywords = ["trust, security, notary, deployment"]
+[menu.main]
+parent= "smn_content_trust"
++++
+<![end-metadata]-->
+
+# Deploying Notary
+
+## The easy way
+
+The easiest way to deploy Notary right now is by using Docker Compose.
+
+Please refer to [Install Docker Compose](../../../compose/install.md) before continuing.
+
+1. Clone the Notary repository
+    ```
+    git@github.com:docker/notary.git
+    ```
+2. Build and start Notary with the sample certificates.
+    ```
+    docker-compose up -d
```

Comment:
same here with the code-blocks

(tip: you can check the rendered version by running `make docs`)